src/item.ts

```diff
@@ -90,6 +92,11 @@ type Image = {
     role: Option<Role>;
 }
 
+type Audio = {
+    kind: ElementKind.Audio;
+    html: string;
```

Comment:
Now that we're parsing this into a `DocumentFragment`, can I suggest we simply pull out the iframe URL and change this to:

```suggestion
    src: string;
```

And then in `renderer.ts` we can do:

```ts
interface AudioProps {
    src: string;
}

const audioStyles = css`
    border: none;
`;

const Audio: FC<AudioProps> = ({ src }) =>
    <iframe src={src} sandbox="allow-scripts" height="380" width="300" />
```

I'm not too clear what the rules are for `height` and `width`. If we can't decide on them ourselves and need to derive those from the CAPI response each time, we can pull those out when we grab the `src` from the `iframe`, so that we have:

```ts
// item.ts
type Audio = {
    kind: ElementKind.Audio;
    src: string;
    height: number;
    width: number;
}

// renderer.ts
interface AudioProps {
    src: string;
    height: number;
    width: number;
}
```